lib/components/artist/related_artists/related_artist.js

```diff
@@ -52,6 +49,19 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
+  initialVariables: { imageWidth: 150, imageHeight: 100 },
+  prepareVariables: (prevVariables) => {
+    const {windowHeight, windowWidth} = Dimensions.get('window');
+    let newWidth = (windowWidth - 60)/2;
+    console.log(newWidth);
+    console.log('window width: ' + windowWidth);
```

Comment:
The `windowWidth` here is undefined, but in Facebook's documentation, they're using some sort of window: https://facebook.github.io/relay/docs/api-reference-relay-container.html#preparevariables

🤔